source/network/load-balancers/components/geo-ip.md

```diff
@@ -0,0 +1,43 @@
+# Geo IP
+
+You can use the Geo IP settings on a listener to allow or deny requests from specific countries and continents..
+
+To get started go to the listener you want to change the Geo IP settings for within the [load balancers section](https://my.ukfast.co.uk/load-balancers) of MyUKFast. Then click the `Geo IP Allow/Block List` tab at the top of the screen.
+
+![Geo IP Tab](../files/geo_ip_1.png)
+
+Firstly enable Geo IP on the listener by clicking the toggle at the top of the screen to change the setting to "Enabled".
+
+## List Type
+
+You can toggle the list between an allow and deny using the "List Type" toggle.
+
+* Allow List - Only users in the countries/continents specified will be allowed to access your load balancer
+* Deny List - Any user in a country/continent specified will be blocked from being able to access your load balancer
+
+## EU Filtering
+
+As well as being able to specify specific countries or continents, you can also add all countries within the EU to the list as a group. For instance, if you only want EU customers to be able to access your website you could have an allow list setup with EU filtering included.
```

Comment:
Could be worth adding another sentence about the UK here just for clarity, e.g. "Note that the EU list does not include the United Kingdom".